test/test_fhirstore.py

```diff
@@ -183,20 +219,22 @@ def test_update_resource(self, store: FHIRStore, test_patient):
     def test_patch_bad_resource_type(self, store: FHIRStore):
         """patch() raises if resource type is unknown"""
 
-        with raises(NotFoundError, match='unsupported FHIR resource: "unknown"'):
+        with pytest.raises(NotFoundError, match='unsupported FHIR resource: "unknown"'):
             store.patch("unknown", "864321", {"gender": "other"})
 
     def test_patch_resource_not_found(self, store: FHIRStore, test_patient):
         """patch() returns None when no matching document was found"""
 
-        with raises(NotFoundError):
+        with pytest.raises(NotFoundError):
             store.patch("Patient", test_patient["id"], {"name": "Patator"})
 
+    # TODO
+    @pytest.mark.skip()
```

Comment:
yes, when performing a patch operation, we don't have the entire FHIR resource and therefore validation with fhir.resources does not work. We should fetch the resource first, manually apply the patch and validate the resource for this test to pass. We didn't have the problem before since validation was performed by mongo

Reply:
Hum, right, it's a pain. I opened an issue